service/metrics/metrics.go

```diff
@@ -101,27 +125,20 @@ func NewMetrics() Metrics {
 			Name:      "ticker_inclusion_status",
 			Help:      "The number of times a ticker was included (or not included) in this validator's vote",
 		}, []string{TickerLabel, InclusionLabel}),
-		prepareProposalTime: prometheus.NewHistogram(prometheus.HistogramOpts{
-			Namespace: AppNamespace,
-			Name:      "oracle_prepare_proposal_time",
-			Help:      "The time it took for the oracle-specific parts of prepare proposal",
-			Buckets:   prometheus.ExponentialBuckets(1, 2, 10),
-		}),
-		processProposalTime: prometheus.NewHistogram(prometheus.HistogramOpts{
+		abciMethodLatency: prometheus.NewHistogramVec(prometheus.HistogramOpts{
 			Namespace: AppNamespace,
-			Name:      "oracle_process_proposal_time",
-			Help:      "The time it took for the oracle-specific parts of process proposal",
-			Buckets:   prometheus.ExponentialBuckets(1, 2, 10),
-		}),
+			Name:      "abci_method_latency",
+			Help:      "The time it took for an ABCI method to execute slinky specific logic (in seconds)",
+			Buckets:   []float64{.0001, .0004, .002, .009, .02, .1, .65, 2, 6, 25},
```

Comment:
buckets chosen are in seconds, these are the same buckets used by cometBFTs, abci_method_latency histogram